### PR TITLE
Fix #15452.

### LINKED ISSUE #15452: CompleteMultipartUpload not working when connected to single bucket using non AWS connection profile
Issue for #15449.

### PR DESCRIPTION
**Pull request by dkocher**

### CODE REVIEW

File: profiles/default/Box.cyberduckprofile
Comment (ylangisc) [inline profiles/default/Box.cyberduckprofile:1]: Doesn't seem related.

**Review by ylangisc — CHANGES_REQUESTED** [review]

File: profiles/default/Box.cyberduckprofile
Comment (dkocher) [reply profiles/default/Box.cyberduckprofile:1]: Rebase to `master`

**Review by ylangisc — APPROVED** [review]
